radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/TimestampedECDSASignatures.java

```diff
@@ -111,7 +111,9 @@
 	private Map<BFTNode, TimestampedECDSASignature> nodeToTimestampedSignature;
 
 	@JsonCreator
-	public static TimestampedECDSASignatures from(@JsonProperty("signatures") Map<String, TimestampedECDSASignature> signatures) {
+	public static TimestampedECDSASignatures from(
+		@JsonProperty("signatures") Map<String, TimestampedECDSASignature> signatures
```

Comment:
If we pass a null value this method fails cause of an implicit requireNonNull in the collector. Would we like to add an explicit check in our code?

Reply:
If we pass `null` here, it will be properly handled. Or you mean that here we should check map internals?

Reply:
I meant when we receive a `Map` with `TimestampedECDSASignature` value as null.